Providers/nxOMSAutomationWorker/automationworker/scripts/onboarding2.py

```diff
@@ -280,7 +280,9 @@ def register(options):
                "OperatingSystem": 2,
                "SMBIOSAssetTag": asset_tag,
                "VirtualMachineId": vm_id,
-               "Subject": subject}
+               "Subject": subject,
+               "NotBeforeUtc": not_before,
```

Comment:
Will changing existing payload cause any issue ?

Reply:
no, will test it in EAP